### PR TITLE
Update RNs for K8ssandra 1.4.0

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/release-notes/_index.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ Release date: 27-August-2021
...
 * [Apache Cassandra](https://cassandra.apache.org/) - the deployed version depends on the configured `cassandra.version` setting:
-  * 4.0.0 (default)
+  * 4.0.1 (default)
   * 3.11.11
   * 3.11.10
   * 3.11.9
   * 3.11.8
   * 3.11.7
-* DataStax Kubernetes Operator for Apache Cassandra ([cass-operator](https://github.com/k8ssandra/cass-operator)) 1.7.1
+* [cass-operator](https://github.com/k8ssandra/cass-operator)) 1.8.0.
```
Comment (jdonenine) [inline docs/content/en/release-notes/_index.md:47]: This should go to 1.9.0 before we do the k8ssandra release.

File: docs/content/en/release-notes/_index.md [hunk 2]
```diff
@@ -38,15 +38,15 @@ Release date: 27-August-2021
...
 * Management API for Apache Cassandra ([MAAC](https://github.com/datastax/management-api-for-apache-cassandra)) 0.1.27
-* [Stargate](https://github.com/stargate/stargate) 1.0.29
+* [Stargate](https://github.com/stargate/stargate) 1.0.40
 * Metric Collector for Apache Cassandra ([MCAC](https://github.com/datastax/metric-collector-for-apache-cassandra)) 0.2.0
 * kube-prometheus-stack 12.11.3 [chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
```
Comment (jdonenine) [inline docs/content/en/release-notes/_index.md:51]: We're bumping this up to v20.0.1 -- it's probably worth noting that we are remaining with version 7.5.11 of Grafana specifically to preserve the inclusion of the last ASL version of Grafan: 
 https://github.com/k8ssandra/k8ssandra/pull/1175/files#diff-bb025298fadbc3d3f32ec53fad275576fd87b5ed47e1aa66f6363c56a488df7eR842

File: docs/content/en/release-notes/_index.md [hunk 3]
```diff
@@ -235,6 +87,40 @@ To manually restart Stargate nodes:
...
+
+Release date: 15-November-2021
+
+The following sections summarize and link to key revisions in K8ssandra 1.4.0. For the latest, refer to the [CHANGELOG](https://github.com/k8ssandra/k8ssandra/blob/main/CHANGELOG-1.4.md).
+
+### Changes
+
+* Update to Reaper 3.0.0.
+* Update to Stargate 1.0.40 [#1118](https://github.com/k8ssandra/k8ssandra/issues/1118).
+* Update to cass-operator 1.8.0-rc.1 [#950](https://github.com/k8ssandra/k8ssandra/issues/950).
```
Comment (jdonenine) [inline docs/content/en/release-notes/_index.md:100]: As mentioned above, this isn't available yet, but we'll want to update this to 1.9.0.
We can also mention the upgrade to kube-prometheus-stack here if you want.

**Review by jdonenine — CHANGES_REQUESTED** [review]
Thanks for putting these together.  Just noted a couple of changes we'll want to make as we solidify things for the release.  I'll let ya know when we're made the cass-operator release and bump within K8ssandra.

File: docs/content/en/release-notes/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md]: Updated to anticipate 1.9.0 - thanks.

File: docs/content/en/release-notes/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md:51]: Updated - thanks.

File: docs/content/en/release-notes/_index.md (hunk 3 above)
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md]: Updated to anticipate 1.9.0 - thanks.

**Review by jdonenine — APPROVED** [review]

**Pull request merged**